_posts/2023-03-20-balanco-ODDs-2023.md

```diff
@@ -4,7 +4,7 @@ categories:
     - Open Data Day
     - Eventos
     - Dados Abertos
-author_staff_member: rasrbk
+author_staff_member: ras_rbk
```

Comment:
como o nome do arquivo é rebeca_almeida.md, aqui só funciona se for `staff_member` rebeca_almeida, o "apelido" não serve

```suggestion
author_staff_member: rebeca_almeida
```